lib/matplotlib/axes/_axes.py

```diff
@@ -1603,9 +1603,7 @@ def semilogx(self, *args, **kwargs):
 
         Other Parameters
         ----------------
-        **kwargs :
-            Keyword arguments control the :class:`~matplotlib.lines.Line2D`
-            properties:
+        **kwargs : Keyword arguments control the :class:`~matplotlib.lines.Line2D` properties:
```

Comment:
Please keep the description on its own line (only type information should really be on the same line).